examples/Python/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -31,16 +37,17 @@
               selects=[selects, selects_secret],
               trim_prefixes=["testapp_settings_"])
```

Comment:
To not conflict with the `config` variable name, we can change this to `azure_appconfiguration`.